### ISSUE TITLE
[1.10.2] ore generation - cracked stone and niter

### ISSUE THREAD

**Issue by ruyanve** [WhiskyTangoFawks/Expedition#61]
I only use ore generation and cave decoration and while I kind of like the variety of sometimes seeing cracked stone I don't really understand why it's there in the first place.
Thinking about performance I wonder if the generation of this will eat up a tick or two and I can tell you, I would take a tick over a cracked stone texture without any real use ANY day!
This may even confuse some players, since it shows up as mossy stone in top and waila, looks like cracked stone and drops as regular cobble.

I also was wondering why there is a new resources that I need to edit out in the first place? I feel it's inconsistent to add a new resource in ore generation even if I disable gameplay tweaks (where it belongs).

Don't get me wrong, I understand that it's part of the package and many people will find it useful and fun, it's just my observation while working on a server pack with your mod as the basis for the most important things ;-)

**Comment by WhiskyTangoFawks** [comment]
It's part of the gameplay tweaks, to make mining more interesting.  If you don't like it, disable it.  The mod is designed as a big pack of things to change up minecraft, there are a lot of different features.  Disable the ones you don't like.

**Comment by ruyanve** [comment]
Ok, let me rephrase since I seem to have explained it poorly...

Disabling Gameplay Tweaks does NOT hinder cracked stone and Niter to generate. Why?

I've been using your mods since forever mate, love that stuff, was a little disappointed when you merged, but now I want to use it as a base for a new server.

**Comment by WhiskyTangoFawks** [comment]
The short answer is that it just didn't occur to me to do it, seemed like it would fall under the ores module, which is where it is.

From a programming perspective, it's also a lot easier to have it through the ores, than through the gameplay config.  And a user can easily remove them from the generation string (just delete them from the master ore gen list) if they don't want that config.

And yeah, it should probably be split into a few different mods.  But having done that for 1.7.10, it was a big pain for me to maintain so many different things.  Hence the merger.

**Comment by ruyanve** [comment]
Aye, that's what I thought. All good, like I said, now that I worked a little with it, I actually like the fact that it's in a bundle.
If I can just remove the entries from the master list that's fine with me as well. Let's see how often people complain about it in the future :-D
